CRM/Core/DAO/permissions.php

```diff
@@ -177,6 +177,15 @@ function _civicrm_api3_permissions($entity, $action, &$params) {
     ),
   );
 
+  // Campaign permissions
+  $permissions['campaign'] = array(
+    'default' => array(
+      'access CiviCRM',
+      'manage campaign',
+    ),
```

Comment:
IMHO, users having only 'administer CiviCRM' should not be able to operate Campaign stuffs (Create/Delete them). Also, when I check from UI perspective, user with 'manage campaign' is having a permission to delete. Updated this PR to add these changes. Thanks for the suggestion:)